README.es.md

```diff
@@ -27,7 +27,7 @@
 [KO_KR]:README.ko_kr.md
 [HI_IN]:README.hi_in.md
 
-Código fuente original del ordenador de guía de la Apollo 11 (AGC) para el módulo de comando (Comanche055) y el módulo lunar (Luminary099). Digitalizado por los amigos de [Virtual AGC][3] y [el museo del MIT][4]. El objetivo es ser un repositorio del código fuente original de la Apollo 11. De este modo, las PRs serán bienvenidas para cualquier cuestión que se identifique entre las transcripciones en este repositorio y los escaneos del código fuente original para [Luminary 099][5] y [Comanche 055][6], así como cualquier fichero que se haya olvidado.
+Código fuente original del ordenador guía del Apollo 11 (AGC) para el módulo de comando (Comanche055) y el módulo lunar (Luminary099). Digitalizado por los chicos de [Virtual AGC][3] y [el museo del MIT][4]. El objetivo es ser un repositorio del código fuente original del Apollo 11. De este modo, serán bienvenidos los PRs relacionados con las transcripciones en este repositorio y los escaneos del código fuente original para el [Luminary 099][5] y el [Comanche 055][6], así como cualquier archivo que haya olvidado.
```

Comment:
Would this also be `Apolo` in the first and third sentences?

Reply:
Yep should be.